### PR TITLE
Fixed menu radio icons.

### PR DESCRIPTION
**Pull request by gennariarmando**

### CODE REVIEW

File: src/core/Frontend.cpp [hunk 1]
```diff
@@ -286,6 +286,11 @@ ScaleAndCenterX(float x)
 	CFont::SetScale(MENU_X(MENUHEADER_WIDTH), MENU_Y(MENUHEADER_HEIGHT)); \
 	CFont::SetFontStyle(FONT_HEADING);
 
+#define DrawRadioIcon(sprite, x, y, radioId, hoverOpt) \
+	sprite.Draw(x, y, MENU_X(MENURADIO_ICON_SCALE), MENU_Y(MENURADIO_ICON_SCALE), radioId == m_PrefsRadioStation ? CRGBA(255, 255, 255, 255) : CRGBA(225, 0, 0, 170)); \
```
Comment (erorcun) [inline src/core/Frontend.cpp:290]: MENU_X(menuXYpadding), MENU_Y(menuXYpadding)

File: src/core/Frontend.cpp [hunk 2]
```diff
@@ -880,8 +885,8 @@ CMenuManager::Draw()
 
 	for (int i = 0; i < NUM_MENUROWS; ++i) {
 		if (aScreens[m_nCurrScreen].m_aEntries[i].m_Action != MENUACTION_LABEL && aScreens[m_nCurrScreen].m_aEntries[i].m_EntryName[0] != '\0') {
-			wchar *rightText = nil;
-			wchar *leftText;
+			wchar* rightText = nil;
```
Comment (erorcun) [inline src/core/Frontend.cpp:888]: it seems VS broke the styling like here and added new lines, pls fix them

File: src/core/Frontend.cpp [hunk 3]
```diff
@@ -1234,46 +1242,42 @@ CMenuManager::Draw()
...
-					if (DMAudio.IsMP3RadioChannelAvailable())
-						fMp3Pos = 34.0f;
-
-					if (i < USERTRACK)
-						m_aFrontEndSprites[i + FE_RADIO1].Draw((SCREEN_WIDTH * 0.5) + SCREEN_SCALE_X(-fMp3Pos + MENURADIO_ICON_X + (fIconSpacing * i)), MENU_Y(nextYToUse), MENU_X(menuXYpadding), MENU_Y(menuXYpadding), i == m_PrefsRadioStation ? CRGBA(255, 255, 255, 255) : CRGBA(225, 0, 0, 170));
-					if (i > CHATTERBOX && DMAudio.IsMP3RadioChannelAvailable())
-						m_aMenuSprites[MENUSPRITE_MP3LOGO].Draw((SCREEN_WIDTH * 0.5) + SCREEN_SCALE_X(-fMp3Pos + MENURADIO_ICON_X + (fIconSpacing * i)), MENU_Y(nextYToUse), MENU_X(menuXYpadding), MENU_Y(menuXYpadding), i == m_PrefsRadioStation ? CRGBA(255, 255, 255, 255) : CRGBA(225, 0, 0, 170));
-#endif
-				}
+				float nextIcon = 40.0f;
```
Comment (erorcun) [inline src/core/Frontend.cpp:1266]: we don't need that anymore

**Comment by erorcun** [comment]
looks good

**Comment by Nick007J** [comment]
lgtm

**Pull request merged**